### PR TITLE
[Doc] Add FAQ for integration on the lower minimum deployment target

### PR DESCRIPTION
**Pull request by AmyL219**
## Purpose

* Add FAQ for integration on the lower minimum deployment target

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by vhuseinova-msft — DISMISSED** [review]

File: docs/manual-installation.md [hunk 1]
```diff
@@ -73,6 +73,34 @@ Note: Please refer to [How to fix the issues](#how-to-fix-the-issues) to fix som
...
+    }
+}
+```
+
+
+
+
+
+
+
```
Comment (jimchou-dev) [inline docs/manual-installation.md:106]: Remove these extra lines?

**Review by jimchou-dev — DISMISSED** [review]

File: docs/manual-installation.md [hunk 2]
```diff
@@ -73,6 +73,34 @@ Note: Please refer to [How to fix the issues](#how-to-fix-the-issues) to fix som
 
     1. Update your target scheme with `ENABLE_ADDRESS_SANITIZER = YES`. For more instruction, please visit [How do you enable Clang Address Sanitizer in Xcode?](https://stackoverflow.com/questions/32150924/how-do-you-enable-clang-address-sanitizer-in-xcode). 
     
+### FAQ
+#### How to conditionally use the UI library based on the iOS version when the project has a lower minimum deployment target?
+The UI library wouldn't work for the device target lower than iOS 14. But it's still possible to import the UI library with the following workaround and conditional launch the UI library when iOS 14 is available.
```
Comment (palatter) [inline docs/manual-installation.md:78]:
```suggestion
The UI library won't work for devices lower than iOS 14 but it's still possible to import the UI library with the following workaround and then conditionally launch the UI library when iOS 14 is available. 
```

File: docs/manual-installation.md [hunk 3]
```diff
@@ -73,6 +73,34 @@ Note: Please refer to [How to fix the issues](#how-to-fix-the-issues) to fix som
 
     1. Update your target scheme with `ENABLE_ADDRESS_SANITIZER = YES`. For more instruction, please visit [How do you enable Clang Address Sanitizer in Xcode?](https://stackoverflow.com/questions/32150924/how-do-you-enable-clang-address-sanitizer-in-xcode). 
     
+### FAQ
+#### How to conditionally use the UI library based on the iOS version when the project has a lower minimum deployment target?
+The UI library wouldn't work for the device target lower than iOS 14. But it's still possible to import the UI library with the following workaround and conditional launch the UI library when iOS 14 is available. 
+
+1. Add a framework to your project’s target and set the minimum deployment target > iOS 14. 
+	* Use this framework for all needed dependencies import, through Cocoapods or manual embed.
```
Comment (palatter) [inline docs/manual-installation.md:81]:
```suggestion
	* Use this framework for all needed dependencies imported through Cocoapods or manual embed.
```

File: docs/manual-installation.md [hunk 4]
```diff
@@ -73,6 +73,34 @@ Note: Please refer to [How to fix the issues](#how-to-fix-the-issues) to fix som
...
+	* Create an UIViewController only targeted for the framework.
+	* Import needed dependency modules inside the UIViewController, add `tokenCredential`, `callCompositeOptions`, and code in the `init()` method to launch the UI library. 
+	* Set this UIViewController as the `NSPrincipalClass` for the newly added framework. 
+	* If needed, the principal class could play a complex role like returning different classes by implementing the protocol from the main target.
+2. Embed the newly added framework into your main target's "Frameworks and Libraries" section with `Embed and Sign`.
+3. In main app, use the [`Loadable Bundles in Cocoa`](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingCode/Concepts/CocoaBundles.html#//apple_ref/doc/uid/20001269-BAJCIAHA) to load the new added UIViewController from the framework when the iOS 14 is available. Code example:
```
Comment (palatter) [inline docs/manual-installation.md:87]:
```suggestion
3. In main app, use the [`Loadable Bundles in Cocoa`](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingCode/Concepts/CocoaBundles.html#//apple_ref/doc/uid/20001269-BAJCIAHA) to load the newly added UIViewController from the framework when the iOS 14 is available. Code example:
```

**Review by palatter — DISMISSED** [review]
Looks good, just some minor suggestions on the wording.

**Comment by AmyL219** [comment]
> Looks good, just some minor suggestions on the wording.

Thanks for rewording!

**Review by jimchou-dev — DISMISSED** [review]

**Review by kevinyulu — DISMISSED** [review]

**Review by palatter — DISMISSED** [review]

**Review by palatter — APPROVED** [review]

**Review by jimchou-dev — APPROVED** [review]

**Pull request merged**